hibernate-core/src/main/java/org/hibernate/dialect/H2Dialect.java

```diff
@@ -101,6 +101,12 @@ public H2Dialect(int version, int buildId) {
 			this.sequenceInformationExtractor = SequenceInformationExtractorNoOpImpl.INSTANCE;
 			this.querySequenceString = null;
 		}
+
+		if ( version < 200 ) {
```

Comment:
Shouldn't this be `<=` instead? Current version of h2 is 1.4.200.

Reply:
I don't think so, wasn't this the thing we all agreed that they changed in the .200 release?

Reply:
By "changed" I mean fixed.